src/Context/ContextApi.php

```diff
@@ -296,4 +297,20 @@ public function renderContext($contextUid)
         return $this->sendRequest($endpoint, $requestData, self::HTTP_METHOD_POST);
     }
 
+    /**
+     * Update resource state.
+     *
+     * @param $resourceId
+     * @param UpdateResourceStateParameters $params
+     * @return bool
+     * @throws SmartlingApiException
+     */
+    public function updateResourceState($resourceId, UpdateResourceStateParameters $params)
+    {
+        $endpoint = \vsprintf('resources/%s/state', $resourceId);
```

Comment:
Should be one of
```
    $endpoint = \vsprintf('resources/%s/state', [$resourceId]);
```
(vsprintf expects second argument to be array, will fail with "vsprintf(): Argument #2 ($values) must be of type array, string given" if strict types are enabled or with later php versions)
```
   $endpoint = \sprintf('resources/%s/state', $resourceId);
```
or
```
     $endpoint = "resources/$resourceId/state";
```

Reply:
Thanks, done